airbyte-integrations/connectors/source-kobotoolbox/source_kobotoolbox/spec.yaml

```diff
@@ -0,0 +1,21 @@
+documentationUrl: https://docs.airbyte.com/integrations/sources/kobotoolbox
+supportsNormalization: false
```

Comment:
It isn't relevant to sources remove it.

Reply:
@marcosmarxm is there a way to remove/disable the normalization option from the UI. This source does not support it.